os/gcache/gcache_adapter_redis.go

```diff
@@ -39,7 +39,7 @@ func (c *AdapterRedis) Set(ctx context.Context, key interface{}, value interface
 		if duration == 0 {
 			_, err = c.redis.Set(ctx, redisKey, value)
 		} else {
-			err = c.redis.SetEX(ctx, redisKey, value, int64(duration.Seconds()))
+			_, err = c.redis.Do(ctx, "SET", redisKey, value, "PX", duration.Milliseconds())
```

Comment:
Please use `Set` function directly instead of using `Do`. https://github.com/gogf/gf/blob/7c92c2f7e85b738a48f26ebd12ac1302d5ac95f7/database/gredis/gredis_redis_group_string.go#L18

Reply:
👌